crates/eframe/src/web/text_agent.rs

```diff
@@ -34,6 +34,7 @@ impl TextAgent {
         style.set_property("height", "1px")?;
         style.set_property("caret-color", "transparent")?;
         style.set_property("position", "absolute")?;
+        style.set_property("display", "none")?;
```

Comment:
Please adda comment explaining why we have this line of code here, so we don't accidentally remove it again!

Reply:
Will do, it seems we didn't remove it since the last time anyway. The issue was previously with the canvas, this time it is the input we add.

I am going to test it a little bit more today, when added as a "home screen app" on Android, I noticed the behavior popped up again somehow, I want to confirm if that was an older cached version or another corner case.

Reply:
Plot thickens: https://chromestatus.com/feature/5093566007214080

So the `display: none` fixes the issue on non-touch devices. On touch devices we still seem to have the same thing happening with the following coinciding with it, trying without the events added now.

Reply:
So it seems if touch events are not enabled (via developer tools or by actually being on a touch-capable device) when the app is getting loaded, but afterwards, this doesn't happen at all.

The `display: none` fix seems to be effective to have this issue not happen on non-touch devices.
But with or without `display: none`, it continues to happen on touch devices, even with zero user interaction. The bug that's addressed by `display: none` originates from the `TextAgent` inputbox and does not show its head unless user interaction takes place. However the other one seems to happen regardless of user interaction.

Will continue to investigate. I think we should merge this (with my added comment soon) and I will try to see where the non-user interaction requiring version of this behavior stems from.